### PR TITLE
Changed dateUtil.service

### PR DESCRIPTION
**Pull request by sendilkumarn**
check this one before merge.

### CODE REVIEW

File: generators/client-2/templates/src/main/webapp/app/components/util/_date-util.service.ts [hunk 1]
```diff
@@ -1,50 +1,34 @@
...
-    'use strict';
+import { DatePipe } from "@angular/common";
 
-    angular
-        .module('<%=angularAppName%>.common')
-        .factory('DateUtils', DateUtils);
+export class DateUtils {
 
-    DateUtils.$inject = ['$filter'];
+    constructor (private pipe:DatePipe) {}
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/util/_date-util.service.ts]: maybe name the variable as `datePipe`

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/components/util/_date-util.service.ts]: saw that !!! but angular 2 prefers this way... I dunno why... Will change that

File: generators/client-2/templates/src/main/webapp/app/components/util/_truncate-characters.filter.ts [hunk 2]
```diff
@@ -1,37 +1,28 @@
...
-        function charactersFilter(input, chars, breakOnWord) {
-            if (isNaN(chars)) {
-                return input;
-            }
-            if (chars <= 0) {
-                return '';
-            }
-            if (input && input.length > chars) {
-                input = input.substring(0, chars);
+export class characters {
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-characters.filter.ts]: you have to register this as a pipe https://angular.io/docs/ts/latest/guide/pipes.html

File: generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts [hunk 3]
```diff
@@ -1,28 +1,19 @@
...
-            if (words <= 0) {
-                return '';
-            }
-            if (input) {
-                var inputWords = input.split(/\s+/);
-                if (inputWords.length > words) {
-                    input = inputWords.slice(0, words).join(' ') + '...';
-                }
-            }
+export class words {
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts]: this also as pipe

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts]: thought of it... we can actually merge this two into one pipe.

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts]: Do you want to change the file name ?

Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts]: yes to `*.pipe.ts`

Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/util/_truncate-words.filter.ts]: and lets keep one feature per file as per the guidelines

**Comment by deepu105** [comment]
dont forget to rename the files in tests and in `index.js`

**Comment by sendilkumarn** [comment]
tests???

**Pull request merged**